.github/build.in.yml

```diff
@@ -133,8 +133,14 @@ jobs:
         run: |
           rm -rf .lake/build/lib/Mathlib/
           # Fail quickly if the cache is completely cold, by checking for Mathlib.Init
-          lake exe cache get Mathlib.Init
-          lake build --no-build Mathlib.Init && lake exe cache get || echo "No cache for 'Mathlib.Init' available"
+          lake exe cache get Mathlib/Init.lean ## file name
+          if lake build --no-build Mathlib.Init  ## with the file path get "error: unknown package `Mathlib`"
```

Comment:
I would phrase this comment more like "need to specify a module name, not a file path" or something. (If "module name" is indeed the correct term, about which I'm not 100% sure.)

Reply:
This was mostly intended as a test PR, but it only succeeeded in the failing part of the test.  I was hoping to get `cache` to work, but I do not think that this is something that can be fixed with a small tweaking of CI.